image-types/rhel/amazon-ec2.md

```diff
@@ -0,0 +1,94 @@
+# Amazon EC2 Image
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which only be created inside
+EC2. An image in a standard format (ova, vmdk, vhd/x, or raw) must be imported
+from S3 storage. **osbuild-composer** generates this image type in the VHDX
+format, because it is fairly modern, widely used, and uses less space than RAW
+images.
+
+## Architectures
+
+Only `x86_64` images are generated.
+
+## Size
+
+The default image size is 6 GB.
```

Comment:
Are there a maximum or minimum size? If there are output types that would, then the definition should either state what those values are or state that there is no max or min size (so that it's obvious it wasn't just forgotten)

Ideally, this information would be passed to the UI so that we can avoid image sizes that we know won't work.

Reply:
I've clarified a bit by saying that they can be larger when the packages don't fit. In general, we should treat it as: minimum size is the size of the installed packages (+ 10% or so) and there is no maximum size other than what's inherent in the file formats (which is ridiculously huge).